.gitignore

```diff
@@ -0,0 +1,31 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
```

Comment:
- Kindly include ` /node_modules` in your `.gitignore` file. You pushed the `node_modules` directory into your GitHub repository and it is causing your PR to be slow and lagging due to the increased repository size.

![image](https://user-images.githubusercontent.com/38088818/177399723-1d2db73f-b3ae-454a-b3ad-373cf9705b73.png)

- Please see to it that it is deleted in your remote repository for optimal performance.

Reply:
Hi @KDlamini  I have did it